pkg/graph/schema/types/mto_solution.graphql

```diff
@@ -39,12 +39,9 @@ type MTOSolution {
 input MTOSolutionChanges @goModel(model: "map[string]interface{}") {
   name: String # If sourced from a Common Solution, this will resolve from the common solution table (you can't edit this for common solution!)
   facilitatedBy: MTOFacilitator
```

Comment:
note that this is an array, but I think we discussed I will handle in my common solution update PR. If it blocks Gary before that PR is merged, I can work to get him unblocked.